README.md

```diff
@@ -76,12 +78,12 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
 
 - For RPM based systems (RedHat, Oracle, CentOS, SuSE):
 ```
-sudo rpm -Uvh ./omi-1.4.1-0.ssl_100.ulinux.x64.rpm
+sudo rpm -Uvh ./omi-MAJOR.MINOR.PATCH-BUILD.ssl_100.ulinux.x64.rpm
```

Comment:
That's fine, but do you want to give an example for some specific version of OMI (say v1.3.0-2)? This is not to keep up to date, just an example. Like:

For example, if you wish to install the RPM version of OMI for v1.3.0-2, you would use a command like:

```
sudo rpm -Uvh ./omi-1.3.0-2.ssl_100.ulinux.x64.rpm
```

?

Would that be more clear, or do you think what you have is pretty clear?

To be precise, the intent is NOT to keep this version number up to date. It's just an example for a point-in-time release.